experimentation/revolve2/experimentation/optimization/ea/selection/_pareto_frontier.py

```diff
@@ -0,0 +1,73 @@
+from itertools import combinations
+from typing import TypeVar
+
+import numpy as np
+from numpy.typing import NDArray
+
+TValues = TypeVar("TValues")
+TOther = TypeVar("TOther")
+
+
+def pareto_frontier(
+    frontier_values: list[list[TValues]], frontier_order: list[bool], to_take: int
+) -> list[int]:
+    """
+    Return individuals based on their respective frontier values and their domination order.
+
+    For mor information on the pareto frontier check: https://en.wikipedia.org/wiki/Pareto_front.
+
+    :param frontier_values: Lists of values that are used for the frontier. The order of the list represents the importance of a value in descending order. These values need to be numeric.
+    :param frontier_order: List of orders for the values used in frontier selection. True = Descending, False = Ascending.
+    :param to_take: The amount of individuals to return from the frontier.
+    :returns: The index of the individuals that were selected.
```

Comment:
is there a specific ordr to the returned individuals?

Reply:
yes i added the order into the docs. (descending)